BaseProject#1/SOEN341InstagramReplica/SOEN341InstagramReplica/Controllers/UsersController.cs

```diff
@@ -270,5 +304,85 @@ public ActionResult FollowOrUnfollowUser(int newFollowStatus, int loggedInUser,
             return Json(new { user, Status = "Ok", Error = "" });
         }
 
+        public ActionResult LinkAccount()
+        {
+            return View();
+        }
+
+        [HttpPost]
+        public ActionResult LinkAccount(User model)
```

Comment:
The user is not supposed to be able to link the same account.
For example, if I am using my account "bob1", then i cannot link with "bob1". It must be another account.

Reply:
your right, will fix